releasenotes/notes/add-openshift-profile.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: istioctl
+releaseNotes:|
```

Comment:
What's here right now isn't valid yaml so the release notes test is failing.

```
releaseNotes: 
- |
```

Reply:
resolved.